ChartIQ/finsemble-fdc3
# Test component

Updating the test component styling

Testing Steps: 

1. Run the watch script to add FDC3 and the tester to the seed project
1. Load the tester and confirm the bellow items

- [ ] Tester is in the launcher menu
- [ ] When pinned to the Toolbar the tester should display an FDC3 icon
- [ ] The tester adapts to the change in component size
- [ ] Each FDC3 API is covered
- [ ] Menu appears when clicking on the hamburger icon and takes you to the correct location
Test each API call by confirming the following: 
- [ ] If the API has a clipboard icon it copies an example that can be pasted into the input below it
- [ ] If an API has an input box you can input text and the test will update in real-time in the snippet area below 
- [ ] When the run button is clicked the show code result button is no longer disabled (there are one or two calls that do not return values and will remain disabled) 
- [ ] If the show result button is enabled it should show the returned value/output of code action

**Notes:** 
addIntentListener does not seem to be working correctly (tested in the javascript console too)
